unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1992,6 +1992,43 @@ notifyUser dir = \case
   DownloadedEntities n -> pure (P.wrap ("Downloaded" <> P.num n <> "entities."))
   UploadedEntities n -> pure (P.wrap ("Uploaded" <> P.num n <> "entities."))
   NotImplementedYet message -> pure (P.wrap ("Not implemented:" <> P.text message))
+  DraftingRelease branch ver ->
+    pure $
+      P.wrap ("😎 Great! I've created a draft release for you at " <> prettySlashProjectBranchName branch)
+        <> "."
+        <> P.newline
+        <> P.newline
+        <> P.wrap ("While preparing we suggest you create a releaseNotes Doc. It'll automatically show up on Unison Share when you publish.")
```

Comment:
```suggestion
        <> P.wrap ("You can create a " <> P.group (P.backticked "ReleaseNotes : Doc") <> "in this branch to give an overview of the release. It'll automatically show up on Unison Share when you publish.")
```

Reply:
Oops, not sure how I missed this before merging

Reply:
And Github doesn't give me a way to move this forward to a new commit